cylc/rose/stem.py

```diff
@@ -17,28 +17,24 @@
 # You should have received a copy of the GNU General Public License
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
-"""rose stem [path]
+"""rose stem [options] [path]
 
-Install a rose-stem suite using cylc install.
+Install a rose stem workflow with a specified set of source tree(s).
+This command acts as a wrapper to "cylc install" by defining a number of
+additional Jinja2 variables.
 
-To run a rose-stem suite use "cylc play".
-
-Install a suitable suite with a specified set of source tree(s).
-
-Default values of some of these settings are suite-dependent, specified
-in the `rose-suite.conf` file.
-
-Examples
-
-    rose stem --group=developer
-    rose stem --source=/path/to/source --source=/other/source --group=mygroup
-    rose stem --source=foo=/path/to/source --source=bar=fcm:bar_tr@head
+The path to the workflow source directory can be specified using PATH.
+Otherwise it must be provided in a directory named
```

Comment:
"path" appears in lower case in the usage string so `PATH` is likely to be mistaken for `$PATH`:

```suggestion
The path to the workflow source directory can be specified using the
"path" argument. Otherwise it must be provided in a directory named
```